src/main/java/solvd/laba/ermakovich/hf/kafka/consumer/KafkaConsumer.java

```diff
@@ -17,17 +17,24 @@
 @Component
 public class KafkaConsumer {
 
-    private final KafkaReceiver<String, IntegrationEvent> receiver;
+    private final KafkaReceiver<String, EventRoot> receiver;
     private final AccountEventService accountEventService;
-    private final MessageMapper<EventRoot, IntegrationEvent> messageMapper;
+    private final MessageMapper<EventRoot, EventRoot> messageMapper;
+    private final KafkaProducer kafkaProducer;
```

Comment:
@kermakovich your class has a very strange set of deps.

in `KafkaConsumer` (custom class) you have `Receiver<>` and at the same time KafkaProducer(also custom class)
So high-level KafkaConsumer has a dependency of high-level KafkaProducer

it's characteristic of a bad design